commands/build/lib/build.js

```diff
@@ -123,6 +154,7 @@ const config = ({
               })
             : false,
         ],
+        ...carbonCopy,
```

Comment:
Is this necessary to have in this build? Or could it be done by the one building the chart outside of nebula.js?
We do similar "automatic" copy operations (for convenience) in some repos when building, but that's handled outside of nebula.js... see https://github.com/qlik-oss/sn-scatter-plot/blob/main/scripts/copy-ext.js for instance.

It might be a little different when developing for react native though, so I might miss some aspect of it...

Reply:
The idea is that we want to have a workflow that doesn't require the user to think too much.  For example, I do a nebula --build --reactNative and I get a package in my react-native folder, without any more interference.   But, the copy is not necessary and we can have the build command output to the target folder.

Reply:
Done!